secret-hitler-online-interface/src/App.js

```diff
@@ -431,11 +431,12 @@ class App extends Component {
     };
 
     shouldJoinButtonBeEnabled() {
-        return (this.state.joinLobby.length === LOBBY_CODE_LENGTH) && (this.state.joinName.length !== 0)
+        return this?.state?.joinLobby?.length === LOBBY_CODE_LENGTH && 
+            this?.state?.joinName?.length !== 0;
     }
 
     shouldCreateLobbyButtonBeEnabled() {
-        return (this.state.createLobbyName.length !== 0);
+        return this?.state?.createLobbyName.length !== 0;
```

Comment:
This line currently results in the Create Lobby button always being enabled.
Changing it back to `return this.state.createLobbyName.length !== 0` and adding the line `this.shouldCreateLobbyButtonBeEnabled = this.shouldCreateLobbyButtonBeEnabled.bind(this);` in the constructor appears to fix this issue.